src/libraries/System.Diagnostics.StackTrace/tests/StackTraceTests.cs

```diff
@@ -300,6 +304,71 @@ public void ToString_NullFrame_ThrowsNullReferenceException()
             Assert.Equal(Environment.NewLine, stackTrace.ToString());
         }
 
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void ToString_ShowILOffset()
+        {
+            string AssemblyName = "ExceptionTestAssembly.dll";
+            string SourceTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, AssemblyName);
+            string regPattern = @":token 0x([a-f0-9]*)\+0x([a-f0-9]*)";
+
+            // Normal loading case
+            RemoteExecutor.Invoke((asmPath, asmName, p) =>
+            {
+                AppContext.SetSwitch("Switch.System.Diagnostics.StackTrace.ShowILOffsets", true);
```

Comment:
I guess not needed any more? And same in other test

Reply:
Currently, the default value is true, but it can be changed.
I think it is necessary to setup test environment.